examples/astro/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "jsxImportSource": "react",
+    "module": "es2020",
```

Comment:
whoops

Reply:
Why?

Reply:
@zJaaal @fmontes Because of this: https://docs.astro.build/en/guides/environment-variables/#getting-environment-variables